README.md

```diff
@@ -11,19 +11,44 @@ and manage the Falcon sensor on your Kubernetes cluster of choice.
 Operators are packaging methodologies for Kubernetes. Which methodology you use
 is dependent on what the Kubernetes vendor and implementation supports.
 
+# Kubernetes Cluster Compatability
+
+The Falcon Helm chart has been tested to deploy on the following vendor Kubernetes clusters:
+
+* Azure Kubernetes Service (AKS) - Linux Nodes Only
+* Rancher K3s
+  * Nodes must be the following Linux distributions:
```

Comment:
Can we link to the Customer Portal which houses the supported Linux distro list? This way we don't have to worry as things get added/updated/removed

Reply:
done